zanata-war/src/main/java/org/zanata/action/VersionHome.java

```diff
@@ -404,35 +400,31 @@ public void copyValidationFromProject() {
     @Override
     @Restrict("#{s:hasPermission(versionHome.instance, 'update')}")
     public String update() {
-        String state = super.update();
+        String state = updateFromSuper();
         Events.instance().raiseEvent(PROJECT_ITERATION_UPDATE, getInstance());
         return state;
     }
 
     /**
-     * This is for autocomplete components of which ConversationScopeMessages
-     * will be null
+     * Flush any changes made to the managed entity instance to the underlying
+     * database. If the update is successful, a log message is printed and a
+     * transaction success event raised. No FacesMessages are enqueued.
+     *
+     * Copied from {@link org.jboss.seam.framework.EntityHome#update()} to
+     * avoid adding an unwanted "successfully updated" message.
      *
-     * @param conversationScopeMessages
-     * @return
+     * @see org.jboss.seam.framework.EntityHome#update()
+     * @return the String "updated"
      */
-    private String update(ConversationScopeMessages conversationScopeMessages) {
-        if (this.conversationScopeMessages == null) {
-            this.conversationScopeMessages = conversationScopeMessages;
-        }
-        return update();
+    @Transactional
+    public String updateFromSuper() {
+        this.joinTransaction();
+        this.getEntityManager().flush();
+        this.raiseAfterTransactionSuccessEvent();
+        return "updated";
     }
```

Comment:
@carlosmunoz does this look reasonable to you? The code is copied directly from `super.update()` but leaves out the line that adds the "successfully updated" message.